app-shells/autojump/autojump-22.2.4-r2.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
+
+inherit python-r1 python-utils-r1 vcs-snapshot
+
+DESCRIPTION="change directory command that learns"
+HOMEPAGE="https://github.com/joelthelion/autojump"
+SRC_URI="https://github.com/joelthelion/${PN}/archive/release-v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~ppc ~ppc64"
+IUSE="bash-completion python test zsh-completion"
+
+# Not all tests pass. Need investigation.
+RESTRICT="test"
+RDEPEND="bash-completion? ( >=app-shells/bash-4 )
+	python? ( ${PYTHON_DEPS} )
+	zsh-completion? ( app-shells/zsh app-shells/gentoo-zsh-completions )"
+DEPEND="test? ( dev-python/flake8 dev-python/tox )"
+
+src_prepare() {
+	sed -e "s: \(/etc/profile.d\): \"${EPREFIX}\1\":" \
+		-i bin/autojump.sh || die
+}
+
+src_compile() {
+	true
+}
+
+src_install() {
+	dobin bin/autojump
+
+	insinto /etc/profile.d
+	doins bin/${PN}.sh
+
+	insinto /usr/share/"${PN}"/
+	if use bash-completion ; then
+		doins bin/${PN}.bash
```

Comment:
I've changed it to `newbashcomp` in 3c5fbd4. I'm not a bash user, so I can't test it unfortunately.